back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -34,6 +34,7 @@
       parameter :folder, "Filter by folder (project folder id)", required: false
```

Comment:
I surprised this parameter is not conditioned by `CitizenLab.ee?` since folders are a commercial feature.

Reply:
I'd have to use a block, like this
```
if Citizenlab.ee?
  parameter :folder, "Filter by folder (project folder id)", required: false
end
```
Because ...
```
parameter :folder, "Filter by folder (project folder id)", required: false if Citizenlab.ee?
```
would apply the conditional to the `required` statement, right?

Reply:
Done.